### ISSUE TITLE
 over-fitting

### ISSUE THREAD

**Issue by AbnerAi** [milesial/Pytorch-UNet#67]
Hi, happy weekend！
      After training, the probability of the verification set reached 93%, but I found that the predicted result is not as good as 80% of the verification set result. I suspect that it is over-fitting. Can predict.py display the loss value of the predicted result? 
     Looking forward to your reply !

**Comment by milesial** [comment]
Hi, yes once you have the predicted mask :
https://github.com/milesial/Pytorch-UNet/blob/e2e46ce509382a45b1db4e1f639aeed568f6cb3e/predict.py#L150

You can compute the loss with your true masks just like in the training

https://github.com/milesial/Pytorch-UNet/blob/e2e46ce509382a45b1db4e1f639aeed568f6cb3e/train.py#L80

You would have to manually load the true mask and flatten it.

Check out the evaluation too in `eval.py` , you could replace the Dice coefficient with any loss you want.
